package.json

```diff
@@ -36,7 +36,7 @@
     "through2": "^2.0.0"
   },
   "devDependencies": {
-    "ava": "^0.0.4",
+    "ava": "^0.4.2",
```

Comment:
We prefer to use `*` as an AVA version in all imagemin projects. https://github.com/imagemin/imagemin/blob/c1df2f6a65d42c0ecbc472203cb594872f47f315/package.json#L52